proto/booking/pricing/untaxed_reservation_quote.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package holms.types.booking.pricing;
+option csharp_namespace = "HOLMS.Types.Booking.Pricing";
+option go_package = "booking/pricing";
+
+import "booking/pricing/untaxed_reservation_nightly_quote.proto";
+import "booking/pricing/untaxed_incidental_reservation_nightly_quote.proto";
+import "booking/indicators/reservation_indicator.proto";
+
+message UntaxedReservationQuote {
```

Comment:
General comment on a lot of the nomenclature used here: you might want to use the phrase "Pretax" rather than "Untaxed". To me, "untaxed" means, "this won't have tax applied" whereas "pretax" treats the application of tax as more of a "process step", which I think more closely models how this works.